src/pages/ErrorPage/NotFoundPage.js

```diff
@@ -13,12 +13,12 @@ const defaultProps = {
 };
 
 // eslint-disable-next-line rulesdir/no-negated-variables
-function NotFoundPage({onBackButtonPress}) {
+function NotFoundPage(props) {
```

Comment:
Props destructuring can be returned after this page will be migrated to TS.

Reply:
@blazejkustra With destructuring it gives next error
![image](https://github.com/Expensify/App/assets/23176449/f995320d-2d2f-43c8-87bb-c517f52a5d03)
in this line:
https://github.com/Expensify/App/blob/18e25a51a1907f1e34c3667e4cea635714c7c2a7/src/libs/Navigation/AppNavigator/AuthScreens.tsx#L312

I can migrate NotFoundPage to TS, though it will add more updates to this PR, which is already a big one.

Reply:
Thanks for the context @VickyStash! 

> I can migrate NotFoundPage to TS, though it will add more updates to this PR, which is already a big one.

Let's not do it in this PR.